docs/howto/index.md

```diff
@@ -4,9 +4,13 @@
 ```{toctree}
 :maxdepth: 2
 
+Manage charms <manage-charms>
```

Comment:
The intention is that this is roughly:

1. Overall intro.
2. Things to keep in mind throughout development.
3. Things you'll need to do for core functionality.
4. Things to make the charm nicer.